python/lsst/sims/catUtils/mixins/VariabilityMixin.py

```diff
@@ -226,6 +237,9 @@ def applyMicrolens(self, params, expmjd_in):
 
     @register_method('applyAgn')
     def applyAgn(self, params, expmjd_in):
```

Comment:
None of the 'apply[SpecificVariabilityMethod]' methods have docstrings.  This is clearly a problem, but they're also not (in my opinion) designed to be easily called outside of the "pass a varParamStr to applyVariability()" model (i.e. they all just take a dict 'params' without specifying its contents).  I think we should make another ticket to refactor these methods into something that can be called from applyVariability() and something that can be called by hand outside of InstanceCatalogs.